packages/generator/src/files/generate-util.ts

```diff
@@ -67,11 +67,11 @@ const getLocalesTranslationRowSync = (locale: Locale, baseLocale: string): strin
 	${wrappedLocale}${postfix},`
 }
 
-const getSyncCode = ({ baseLocale, locales }: GeneratorConfigWithDefaultValues) => {
+const getSyncCode = ({ baseLocale, locales, esmImports }: GeneratorConfigWithDefaultValues) => {
 	const localesImports = locales
 		.map(
 			(locale) => `
-import ${sanitizeLocale(locale)} from './${locale}'`,
+import ${sanitizeLocale(locale)} from './${locale}${esmImports ? '.js' : ''}'`,
```

Comment:
I think we could improve this by creating a helper function inside `output-handler.ts`
```typescript
export const relativeFileImportParth = (file) => './${file}${esmImports ? '.js' : ''}'
```
and then call it like this
```typescript
relativeFileImportParth(locale)
```

Reply:
Yes, good idea.